TCAT/Views/RouteDiagramSegment.swift

```diff
@@ -143,7 +147,7 @@ class RouteDiagramSegment: UIView {
         testStopLabel.font = .getFont(.regular, size: 14.0)
         testStopLabel.textColor = Colors.primaryText
         testStopLabel.text = name
-
+        testStopLabel.sizeToFit()
```

Comment:
We shouldn't need this

Reply:
JK seems likes we may need this cuz of the jank stuff we do in setting the label text using some fake test label. If u waaaaant u could look into it but not necessary for now

Reply:
Maybe I'll look into this later!